integration/testdata/init/helm-project/skaffold.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: skaffold/v3alpha1
+kind: Config
+metadata:
+  name: helm-project
+build:
+  artifacts:
+  - image: go-hello
+    context: hello-world/src
+    docker:
+      dockerfile: Dockerfile
+deploy:
```

Comment:
Question: is there a way to generate the version of schema with a `manifests` field as well?  Is that something we think users might want?